wavefront_sdk/common/utils.py

```diff
@@ -101,6 +101,35 @@ def is_blank(string):
     # return len(re.sub(r'[\s]+', '', s)) == 0
 
 
+def validate_tags(tags):
+    """
+    Ensure that the tag is not empty, otherwise throw the error.
+
+    :param tags:
+    :return:
+    """
+    for tag in tags:
+        if is_blank(tag):
+            raise ValueError('Event tag: ' + tag + ' cannot be blank')
```

Comment:
My bad. If the tag is empty, logging it (blank string) does not help. This can just be:
`Event tag cannot be blank`.